fasta/assemblyStats_test.go

```diff
@@ -29,3 +32,32 @@ func ContigAllAreEqual(input []int, expected []int) bool {
 	}
 	return true
 }
+
+func TestCalculateGenomeLength(t *testing.T) {
+	input := MakeContigList(inputFasta, false)
+	if calculateGenomeLength(input) != 49 {
+		t.Errorf("Error in calculateGenomeLength. Expected: 49. Observed: %d", calculateGenomeLength(input))
+	}
+}
+
+func TestCalculateN50L50(t *testing.T) {
+	exp := []int{30, 1}
+	input := MakeContigList(inputFasta, true)
+	sort.Ints(input)
+	N50, L50 := CalculateN50L50(input, calculateGenomeLength(input)/2)
+	if N50 != exp[0] || L50 != exp[1] {
+		t.Errorf("Error in CalculateN50L50: Expected N50: %d L50: %d. Observed N50: %d L50: %d", exp[0], exp[1], N50, L50)
+	}
+}
+
+func TestWriteAssemblyStats(t *testing.T) {
```

Comment:
I think the code would be better organized and easier to test in the future if the fasta package contained single functions to calculate the assembly stats (i.e CalcN50, CalcL50, etc...) and the WriteAssemblyStats function should be moved into the cmd and modified there depending on user needs. The fasta package should contain structs and functions that use the fasta struct and I feel like the WriteAssembly function breaks that pattern

Reply:
Would be good to add a TODO at the very least since the `WriteAssemblyStats()` function was preexisting